saleor/celeryconf.py

```diff
@@ -33,5 +33,11 @@ def setup_celery_logging(loglevel=None, **kwargs):
     ],
     related_name="saleor3_14",
 )
+app.autodiscover_tasks(
+    packages=[
+        "saleor.payment.migrations.tasks",
+    ],
+    related_name="saleor3_14",
+)
```

Comment:
The `autodiscover_tasks` with `saleor3_14` name already exists. You should join it

Reply:
Right. Thanks!